node-scripts/src/pipeline3/scripts.js

```diff
@@ -396,6 +396,24 @@ module.exports = {
       return script;
     },
 
+    /*
+     * Generates a script that pulls a Docker image
```

Comment:
`prepareDeployment` is now a more generic function. It does not necessarily do `docker pull`, so therefore this cannot be its description.
Write instead:

> Generates the script that will prepare the host for deployment